src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java

```diff
@@ -583,10 +592,12 @@ void onBestBlock_whenPegoutTxIsCachedWithInvalidTimestamp_shouldSignSamePegoutTx
 
         // Start first round of execution
         ethereumListener.get().onBestBlock(null, Collections.emptyList());
-    
-        // Ensure the pegout tx becomes invalid by waiting 1 second
-        Thread.sleep(1000);
 
+        // Ensure the pegout tx becomes invalid by advancing the clock 1 hour
+        field = pegoutSignedCache.getClass().getDeclaredField("clock");
+        field.setAccessible(true);
+        field.set(pegoutSignedCache, Clock.offset(baseClock, Duration.ofHours(1)));
```

Comment:
Because we need to change the internal state of the class, in this changing the clock by advancing it by one hour. Notice that the clock instance do not hold the same reference, and the `offset` call return a new reference to a new Clock class.